### DISCUSSION TITLE
bug or implementation issue? - some extensions cannot be installed because latest version of Floorp is supposedly too old?

### DISCUSSION THREAD

**Discussion by Ryder17z** [Floorp-Projects/Floorp#1185] [Q&A]
All I know is that extensions are sometimes in a rocky state. Wasn't aware that this was still a problem as it feels like the last time wasn't that recent.

I'd like to install Enhancer for Youtube on Floorp but the official download link just tells me "You need an updated version of Fireofx to use this extension".

Is this an actual bug or is the implementation of Floorp this far behind?

**Comment by creeper-0910** [comment]
https://github.com/Floorp-Projects/Floorp/discussions/1094

**Reply by Ryder17z** [reply]
thanks.. very helpful
